assets/app/views/create/nextSteps.html

```diff
@@ -0,0 +1,102 @@
+<div ng-controller="ProjectController">
+  <div ng-controller="CreateFinishedController" class="container next-steps">
+    <ol class="breadcrumb">
+      <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
```

Comment:
since you already have it on the scope object, can you use projectDisplayName here, otherwise no reason to have it on the scope